src/pages/en/guides/publish-to-npm.md

```diff
@@ -6,37 +6,42 @@ description: Learn how to publish Astro components to NPM
 
 Building a new Astro component? **Publish it to [npm!](https://npmjs.com/)**
 
-Publishing a component is a great way to reuse work across your team, your company, or the entire world. Astro components can be published to and installed from npm, just like any other JavaScript package.
+Publishing an Astro component is a great way to reuse your existing work across your projects, and to share with the wider Astro-community at large. Astro components can be published directly to and installed from NPM, just like any other JavaScript package.
 
-**Astro’s ability to publish and reuse popular components is one of it’s most powerful features!**
+Looking for inspiration? Check out some of [our favorite themes & components](/en/themes) from the Astro community. You can also [search npm](https://www.npmjs.com/search?q=keywords:astro-component) to see the entire public catalog.
 
-Even if you don't plan on publishing your components online, the patterns outlined below can help any developer design reusable components in isolation from their custom website or business logic.
+> Don't want to go it alone? Check out [Astro Community's component template](https://github.com/astro-community/component-template) for a community-supported, out-of-the-box template!
 
-Looking for inspiration? Check out some of [our favorite themes & components][/themes] from the Astro community. You can also [search npm](https://www.npmjs.com/search?q=keywords:astro-component) to see the entire public catalog.
+## Quick Start
+
+```bash
+npm init astro myComponents -- --template component
```

Comment:
Can you add a `#` comment here to explain what this does?` 

Also, I don't think calling a directory `myComponents` is very common. I might recommend `my-component` or, even better, something that also documents what it is: 

```
npm init astro my-new-component-directory -- --template component
```

also, does `my-new-component-directory` go before the `--`, or after?

Reply:
@delucis @aFuzzyBear Can I please get some feedback on this question re: command line usage here?

Reply:
Perfect sorry I wasn't overly familiar with this nomenclature,